android/src/main/java/com/vonovak/AddCalendarEventModule.java

```diff
@@ -11,9 +11,11 @@
 import android.provider.CalendarContract;
 import android.app.LoaderManager;
 import android.content.Loader;
-import android.support.annotation.Nullable;
 import android.util.Log;
 
+// support for AndroidX
+import javax.annotation.Nullable;
```

Comment:
ah, please use the android X nullable here, not javax, thanks!

Reply:
@vonovak no problem, check new changes thanks